### ISSUE TITLE
Runtime Error when trying to run setup.py

### ISSUE THREAD

**Issue by linuxfold** [aqlaboratory/openfold#130]
After activating the conda environment, and then typing

 `python3 setup.py install`

I get an error:

```
RuntimeError: 
The detected CUDA version (11.7) mismatches the version that was used to compile
PyTorch (10.2). Please make sure to use the same CUDA versions.

```

Do I have to downgrade CUDA to be able to use openfold?

**Comment by gahdritz** [comment]
Hm. If you had a slightly less new CUDA version (say, < 11.4), you could just upgrade your PyTorch installation instead---you can install builds with CUDA 11.3 from the website. That might still be worth a try, but I suspect even the newest one on offer there won't be compatible with CUDA version 11.7. That leaves you with two options: you can downgrade your CUDA or, and this is the better option in my opinion, build PyTorch from source following the instructions here: https://github.com/pytorch/pytorch#from-source.

**Comment by linuxfold** [comment]
I struggled for a few hours trying to build pytorch from source and failed. 

I ended up downloading CUDA 11.3 and changing my .bashrc to point to 11.3 instead of 11.7 (also got rid of reference to /usr/local/cuda). I still have 11.7 installed but not using by default.

I also had to downgrade to gcc8 using 
`sudo update-alternatives --config gcc`

Seems like the setup.py install works now
`Finished processing dependencies for openfold==1.0.0`
